.github/workflows/main.yml

```diff
@@ -365,12 +390,18 @@ jobs:
 
   build-storybook:
     name: Build storybook
+    if: github.event_name != 'merge_group' || needs.check-skip-merge-queue.outputs.skip-merge-queue != 'true'
+    needs:
+      - check-skip-merge-queue
     uses: ./.github/workflows/build-storybook.yml
     secrets:
       STORYBOOK_TOKEN: ${{ secrets.STORYBOOK_TOKEN }}
 
   build-ts-migration-dashboard:
     name: Build ts migration dashboard
+    if: github.event_name != 'merge_group' || needs.check-skip-merge-queue.outputs.skip-merge-queue != 'true'
```

Comment:
I think it might be okay to always skip `build-ts-migration-dashboard` in the Merge Queue (this will also affect some downstream jobs)

Reply:
It's required by `all-jobs-completed` so skipping it would always cause that job to be skipped as well.